app/controllers/provider/admin/registry/policies_controller.rb

```diff
@@ -43,11 +43,21 @@ def create
     end
   end
 
+  def edit; end
```

Comment:
You don't need this I think.

Reply:
I guess code climate will complain without it :/

Reply:
I think you are right and Code Climate should not complain. I will remove

Reply:
Codeclimate complained about it, that was why hehe